### DISCUSSION TITLE
NFT minting dapp publish with v4.2.0

### DISCUSSION THREAD

**Discussion by CryptainGolD** [aptos-labs/aptos-developer-discussions#446] [Questions] [answered]
### Describe your question in detail.

When I am gonna publish nft-minting-dapp on my side, I get this error.
It was working before when the version was 4.1, but it's been upgraded to 4.2 and I keep getting this error message
https://aptos.dev/en/build/create-aptos-dapp/templates/nft-minting-dapp

Anyone from Aptos DEV Community please help me

### What error, if any, are you getting?

Error: Failed to extract object address from output
    at m.extractAddressFromOutput (D:\Work\nft-minting-dapp\node_modules\@aptos-labs\ts-sdk\dist\common\cli\index.js:1:5563)
    at m.createObjectAndPublishPackage (D:\Work\nft-minting-dapp\node_modules\@aptos-labs\ts-sdk\dist\common\cli\index.js:1:3782)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)

Node.js v22.9.0

### What have you tried or looked at?  Or how can we reproduce the error?

Updated node version.

### Which operating system are you using?

Windows

### Which SDK or tool are you using? (if any)

TypeScript SDK

**Comment by 0xmaayan** [comment]
Hi, thanks for reporting! We have identified the issue and opened a PR to address it: https://github.com/aptos-labs/aptos-ts-sdk/pull/513. Once the PR is merged, you can upgrade the `@aptos-labs/ts-sdk` version, and that should resolve the issue.

**Reply by 0xmaayan** [reply, marked as answer]
A new version of the `@aptos-labs/ts-sdk@1.28.0` is out https://www.npmjs.com/package/@aptos-labs/ts-sdk - please upgrade your local dependency version

**Reply by CryptainGolD** [reply]
Thanks for your work. Now it works successfully that set VITE_MODULE_ADDRESS as object address.
And I have another question that how to use candy_machine function to mint collections?
I have a few txns which is occurred from this nft-minting-dapp template, but I can't remember how could I get it.
https://aptoscan.com/transaction/5979610396?network=testnet

**Reply by 0xmaayan** [reply]
This templates uses the smart contract that comes with the template. So once you publish your contract, creating a collection with the template hits the contract you just published, see [here](https://github.com/aptos-labs/create-aptos-dapp/blob/main/templates/nft-minting-dapp-template/frontend/entry-functions/create_collection.ts#L45)

Not completely sure how the candy_machine relates here?

**Reply by CryptainGolD** [reply]
Ah, make sense now. Thank you again
Turning Ether/Sol to Aptos is little complex for me, actually. 😅 
I need your help for Aptos further 🙏 🙏 🙏
